packages/travix-ui-kit/themes/_default.yaml

```diff
@@ -121,6 +121,12 @@ badge:
     right-left: 8px
     top-bottom: 6px
     corner-side: 12px
+  small:
```

Comment:
is it really should be part of a theme?

Reply:
Well, as I see from this file, yes, should be.

Reply:
will it change from brand to brand?